cmd/lncli/commands.go

```diff
@@ -2194,9 +2194,11 @@ func addInvoice(ctx *cli.Context) error {
 	printJSON(struct {
 		RHash  string `json:"r_hash"`
 		PayReq string `json:"pay_req"`
+		Chain  string `json:"chain"`
 	}{
 		RHash:  hex.EncodeToString(resp.RHash),
 		PayReq: resp.PaymentRequest,
+		Chain:  resp.Chain,
```

Comment:
The HRP is not readable. IMHO it is better to be included in the response so that the user will have more understanding.  While it can easily be removed I think we should leave it there.